### PR TITLE
HDDS-9501. Migrate misc. tests in hadoop-ozone to JUnit5

### PR DESCRIPTION
**Pull request by raju-balpande**
## What changes were proposed in this pull request?

Migrate the following test classes to JUnit5. See parent task for details.

hadoop-ozone/insight/src/test/java/org/apache/hadoop/ozone/insight/TestBaseInsightPoint.java
hadoop-ozone/insight/src/test/java/org/apache/hadoop/ozone/insight/TestConfigurationSubCommand.java
hadoop-ozone/insight/src/test/java/org/apache/hadoop/ozone/insight/TestLogSubcommand.java
hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmKeyInfoCodec.java
hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmMultipartKeyInfoCodec.java
hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmPrefixInfo.java
hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmPrefixInfoCodec.java
hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestRepeatedOmKeyInfoCodec.java
hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestS3SecretValueCodec.java
hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestTransactionInfoCodec.java

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-9501

## How was this patch tested?
Manually tested the test cases. Tested it in CI build as well.

### CODE REVIEW

File: hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmKeyInfoCodec.java
Comment (adoroszlai) [inline hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmKeyInfoCodec.java:1]: Please also change assertions:

https://github.com/apache/ozone/blob/05c41392ff63ff0f2d2b7e0f9527dd801a42e46b/hadoop-ozone/interface-storage/src/test/java/org/apache/hadoop/ozone/om/helpers/TestOmKeyInfoCodec.java#L39-L41

File: hadoop-ozone/insight/pom.xml
Comment (adoroszlai) [inline hadoop-ozone/insight/pom.xml:1]: Please also add:

```
  <properties>
    <allow.junit4>false</allow.junit4>
  </properties>
```

File: hadoop-ozone/interface-storage/pom.xml
Comment (adoroszlai) [inline hadoop-ozone/interface-storage/pom.xml:1]: Please also add:

```
  <properties>
    <allow.junit4>false</allow.junit4>
  </properties>
```

**Review by adoroszlai — COMMENTED** [review]
Thanks @raju-balpande for the patch.

File: hadoop-ozone/insight/pom.xml
Comment (raju-balpande) [reply hadoop-ozone/insight/pom.xml:1]: Done.

**Review by adoroszlai — APPROVED** [review]

**Pull request merged**